gui-apps/hyprpaper/hyprpaper-0.7.1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A blazing fast wayland wallpaper utility"
+HOMEPAGE="https://github.com/hyprwm/hyprpaper"
+SRC_URI="https://github.com/hyprwm/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
```

Comment:
I think you should add `gui-libs/hyprutils` as a dependency.
https://github.com/hyprwm/hyprpaper/releases/tag/v0.7.1
> hyprutils: required >=0.2.0

Reply:
Thanks I have updated the dependencies.